bundles/org.openhab.binding.astro/src/main/java/org/openhab/binding/astro/internal/action/AstroActions.java

```diff
@@ -66,7 +66,7 @@ public void setThingHandler(@Nullable ThingHandler handler) {
     }
 
     @RuleAction(label = "get the azimuth", description = "Get the azimuth for a given time.")
-    public @Nullable @ActionOutput(name = "getAzimuth", label = "Azimuth", type = "org.openhab.core.library.types.QuantityType<javax.measure.quantity.Angle>") QuantityType<Angle> getAzimuth(
+    public @Nullable @ActionOutput(name = "result", label = "Azimuth", type = "org.openhab.core.library.types.QuantityType<javax.measure.quantity.Angle>") QuantityType<Angle> getAzimuth(
```

Comment:
@lolodomo Just FYI, it should be possible to leave out the `name` as it now defaults to `result` inside the core code.

Reply:
Correct

Reply:
@lsiepel @florian-h05 : your suggestion is to set name parameters or not ?

Reply:
I would suggest to not set names for actions with a single output.

Reply:
Let's choose to not include name.